concordia/settings_ecs.py

```diff
@@ -87,3 +87,6 @@
     "Transcribed and reviewed by volunteers participating in the "
     "By The People project at crowd.loc.gov."
 )
+
+if os.getenv("USE_PERSISTENT_DATABASE_CONNECTIONS"):
```

Comment:
I'm not sure we want to explicitly test here since we don't otherwise define this value but it might be a good future-proofing move:

```suggestion
if os.getenv("USE_PERSISTENT_DATABASE_CONNECTIONS") == "1":
```